tools/eos-metrics-collector.exe

```diff
@@ -254,6 +256,22 @@ def upload(storage_root):
         sys.exit(msg)
```

Comment:
After making `get_metrics_env` and `get_ostree_url` functions in the global scope, you can do all these new checks here, together with the existing ones.